core-java-modules/core-java-collections-array-list/src/main/java/com/baeldung/listofobjectstolistofstring/User.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.listofobjectstolistofstring;
+
+public class User {
+    private final String userName;
+    private final String firstName;
+    private final String lastName;
+
+    public User(String userName, String firstName, String lastName) {
```

Comment:
The code in the article just passes one `String` to the `User` constructor, maybe we should just simplify this class to be `User(String name)` to make it easy to align to the article?

Reply:
Sure, makes sense.